server/utils/flow/handler.go

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package flow
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+
+	"github.com/mattermost/mattermost-plugin-mscalendar/server/utils"
+	"github.com/mattermost/mattermost-plugin-mscalendar/server/utils/httputils"
+)
+
+type fh struct {
+	flow  Flow
+	store FlowStore
+}
+
+func Init(h *httputils.Handler, flow Flow, store FlowStore) {
+	fh := &fh{
+		flow:  flow,
+		store: store,
+	}
+
+	oauth2Router := h.Router.PathPrefix("/").Subrouter()
+	oauth2Router.HandleFunc(flow.URL(), fh.handleFlow).Methods("POST")
+}
+
+func (fh *fh) handleFlow(w http.ResponseWriter, r *http.Request) {
+	mattermostUserID := r.Header.Get("Mattermost-User-ID")
+	if mattermostUserID == "" {
+		utils.SlackAttachmentError(w, "Error: Not authorized.")
+		return
+	}
+
+	stepNumber, err := strconv.Atoi(r.URL.Query().Get("step"))
+	if err != nil {
+		utils.SlackAttachmentError(w, fmt.Sprintf("Error: Step provided is not an int, err=%s", err.Error()))
+		return
+	}
+
+	step := fh.flow.Step(stepNumber)
+	if step == nil {
+		utils.SlackAttachmentError(w, fmt.Sprintf("Error: There is no step %d.", step))
+		return
+	}
+
+	property := step.GetPropertyName()
+
+	valueString := r.URL.Query().Get(property)
```

Comment:
0/5 remove blank lines in this function like the one after the `property :=` line. I suggest joining it with the next block instead.